### PR TITLE
Fixed Issue_948

### PR DESCRIPTION
**Pull request by Malligarjunan**
Pull Request for #948

### CODE REVIEW

**Comment by Malligarjunan** [comment]
Hello Team,
Please let me know if anything pending on this request to get it merged?

**Comment by Malligarjunan** [comment]
Hello Team,
Please let me know if anything pending on this request to get it merged?

File: modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java [hunk 1]
```diff
@@ -1608,6 +1609,18 @@ public void testIssue931() {
...
     }
 
+    @Test
+    public void testIssue948() {
+        ParseOptions options = new ParseOptions();
+        options.setResolve(true);
+        SwaggerParseResult result = new OpenAPIV3Parser().readLocation("Issue_948.json", null, options);
+        new OpenAPIResolver(result.getOpenAPI()).resolve();
+        assertNotNull(result.getOpenAPI());
+//        assertTrue(result.getMessages().size() > 0);
```
[the remark this answers is not in the document]
Comment (Malligarjunan) [reply modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java]: @gracekarina - Let me know if you have more comments/questions.

[the remark this answers is not in the document]
Comment (gracekarina) [reply modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java]: Yes, please just remove the commented code so I can merge. Thanks!

[the remark this answers is not in the document]
Comment (Malligarjunan) [reply modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java]: @gracekarina  - Removed the commented line and pushed the code changes.

**Pull request merged**